### PR TITLE
Feature/emp3 web#36 implement map.set motion lock mode in leaflet

### LINKED ISSUE #36: Implement Map.setMotionLockMode in Leaflet
This will be need in order for (#13) to work.

### PR DESCRIPTION
**Pull request by nclark25**
resolves #36

### CODE REVIEW

File: src/mapengine/leaflet/js/leaflet-map-engine.debug.js [hunk 1]
```diff
@@ -650,26 +661,24 @@ emp.engineDefs.leafletMapEngine = function (args) {
...
-            overlayId: sCoreId,
-            visible: true,
-            coreId: oControl.getCoreId(),
-            coreParent: sCoreId
-        });
-        oLayerList.push(oLayerEntry);
-
-        instanceInterface.addEmpObject(oControl);
-        instanceInterface.leafletInstance.addControl(oControl);
+        // oControl = new L.Control.Zoom({
```
Comment (jaybarra) [inline src/mapengine/leaflet/js/leaflet-map-engine.debug.js]: Can we remove the commented out code?

File: src/mapengine/leaflet/js/leaflet-map-engine.debug.js [hunk 2]
```diff
@@ -1346,74 +1399,77 @@ emp.engineDefs.leafletMapEngine = function (args) {
...
-                })]);
+                    message: "Overlay " + oEmpObject.options.name + " is empty."
+                  }));
+                  return;
+                }
+              }
+              oEmpBounds = oEmpObject.getFeatureBounds();
+              if (oEmpBounds) {
+                if (bZoom) {
+                  if (oEmpBounds.getCenterWidth() < 16093) {
```
Comment (jaybarra) [inline src/mapengine/leaflet/js/leaflet-map-engine.debug.js]: Can you move the value into a named variable? Could be unclear what 16093 refers to

File: src/mapengine/leaflet/js/leaflet-map-engine.debug.js (hunk 1 above)
Comment (nclark25) [inline src/mapengine/leaflet/js/leaflet-map-engine.debug.js]: Once Jimmy tests the default zoom control that is being used I will remove this code.

File: src/mapengine/leaflet/js/leaflet-map-engine.debug.js (hunk 2 above)
Comment (nclark25) [inline src/mapengine/leaflet/js/leaflet-map-engine.debug.js]: Yes, I will move it.

**Pull request merged**